File: Transloadit/Classes/Resources/Assembly/Assembly.m
//
//  Assembly.m
//  Transloadit
//
//  Created by Mark Masterson on 8/19/16.
//  Copyright © 2016 Mark R. Masterson. All rights reserved.
//

#import "Assembly.h"

@implementation Assembly


- (id)initWithID:(NSString *)assembly_id {
    self = [super init];
    if(self) {
        NSLog(@"_init: %@", self);
        
    }
    return self;
}

- (id)initWithSteps:(AssemblyStep [])steps {
    self = [super init];
    if(self) {
        NSLog(@"_init: %@", self);
        
    }
    return self;
}

-(NSURL)url


@end

File: Transloadit/Classes/Models/Assembly.swift
//
//  Assembly.swift
//  Pods
//
//  Created by Mark Robert Masterson on 11/17/19.
//

import UIKit

class Assembly: APIObject {

    var steps: Array<AssemblyStep>?
    var numberOfFiles: Int?
    
    convenience init(withSteps steps: Array<AssemblyStep>, andNumberOfFiles numOfFiles: Int) {
        self.init()
        self.steps = steps
        self.numberOfFiles = numOfFiles
    }
    
}
